### ISSUE TITLE
Shorten paragraphs instead of streching (=> `\looseness=-1`)

### ISSUE THREAD

**Issue by denismaier** [gucci-on-fleek/lua-widow-control#33]
I usually also try to shorten paragraphs instead of or in addition to making them longer, i.e. I use `\looseness=-1`. Is there any particular reason why this strategy is not used by this package? And, would there be an option to add it?

**Comment by gucci-on-fleek** [comment]
Yeah, I've looked into it and it's usually not really worth it. See https://github.com/gucci-on-fleek/lua-widow-control/issues/4#issuecomment-1039914491 and the [tb133chernoff-widows-extract.pdf](https://github.com/gucci-on-fleek/lua-widow-control/files/8578379/tb133chernoff-widows-extract.pdf) §15.3 for some explanations.

I'm not opposed to adding this though — it's just not a priority. Feel free to submit a PR; otherwise, I'll probably be awhile before I get around to this.

**Comment by denismaier** [comment]
Thanks for the explanation and for pointing me to the relevant discussions. It's not urgent for me either – I was just wondering.
